src/components/ProfileImages/ProfileImagesOptionList.style.js

```diff
@@ -33,4 +33,12 @@ export const ProfileImageOptionItems = styled.li`
   background-repeat: no-repeat;
   background-size: cover;
   cursor: pointer;
+
+  &:hover {
+    border: 0.1rem solid var(--gray-500, #555);
+  }
+
+  &:active {
+    border: 0.2rem solid var(--gray-700, #3a3a3a);
+  }
```

Comment:
까만 테두리가 잘 안보여서 아예 색상 필터를 입히는 건 어떤가용 ..? 개인 취향의 영역이라 그냥 제안드려봅니다.ㅎ..

그리구 까만테두리 잘안보여서 이런식으로 하는 것도 나쁘지 않아보입니당 ㅎㅎ (개인적인 의견이니까 참고만 해주세용 )

<img width="69" alt="image" src="https://github.com/Dev-Duke-Seo/rolling_team1/assets/127701092/d4417c8f-42dd-4b36-b585-efdc621660f7">

```suggestion
  &:hover {
    filter : opacity(0.5) drop-shadow(0 0 0 #861DEE);
  }

  &:hover {
    filter : opacity(0.5) drop-shadow(0 0 0 #5603A7);
  }
```

그리고 궁금한게 클릭하고 나서도 남아있게 할 수는 없나요? 하라는 뜻 아니고 질문입니다!!!!